server/placeSearch/placeSearchController.js

```diff
@@ -71,19 +100,25 @@ module.exports = {
     }));
 
     // Promise.all returns an array of objects, same length as the number of tags.
-    Promise.all(optionsArray.map(options =>
-      rp(options)
-        .then(data => {
-          // We return an object that has all the places (formatted) as the value paired to
-          // a key of the tag name.
-          const places = data.results;
-          const tagObj = {};
-          tagObj[options.tripsAppTag] = places.map(place => formatPlace(place, options));
-
-          return tagObj;
-        })
-        .catch(err => helpers.errorHandler(err, req, res, next))
-    ))
+    Promise.all(optionsArray.map(options => {
+      const key = '' + options.qs.query + options.qs.types; // eslint-disable-line
```

Comment:
Use ES6 syntax: \`${options.qs.query}:${options.qs.types}\`

Let's also separate query and types with a `:`